<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1132</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="yoloTextLayout">
        <item>
         <layout class="QVBoxLayout" name="fopLayoutV">
          <item>
           <layout class="QHBoxLayout" name="browseLayoutH">
            <item>
             <widget class="QPushButton" name="browseButton">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="browseLineEdit">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="extractLayoutH">
            <item>
             <widget class="QPushButton" name="extractButton">
              <property name="text">
               <string>Extract</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="extractLineEdit">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="detectButton">
            <property name="text">
             <string>Detect</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="detectProgress">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="metadataTextEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="videoLabel">
        <property name="minimumSize">
         <size>
          <width>1366</width>
          <height>768</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1132</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>detectButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>detectClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browseButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>browseClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>460</x>
     <y>555</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>extractButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>extractClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>551</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>browseClicked()</slot>
  <slot>extractClicked()</slot>
  <slot>detectClicked()</slot>
 </slots>
</ui>
